content/docs/integrations/github-discovery/index.md

```diff
@@ -22,9 +22,23 @@ This page describes how to do that.
 ## Steps
 
 1. Visit the Administration > Settings page and click on GitHub under Integrations
-2. Find the Add button beside "Configure Autodiscovery of entities" and click the button.
-3. In the dialog the appears enter the name of the url of your GitHub Organization e.g. `https://github.com/your-org`. Press Save
+2. Scroll down to the `Automatically add and remove entities from your Catalog` section.
+   ![add and remove](./add-remove.webp)
+3. Press the `+ ADD ITEM` button in the bottom right corner.
+4. In the input that appears enter a url that you'd like to have roadie to listen for additions and deletions.
+   This url accepts regexes in every segment:
+
+- https://github.com/RoadieHQ/roadie-backstage-plugins/blob/-/catalog-info.yaml // Roadie will only add entities when a catalog-info.yaml file was added/removed to the default branch from the roadie-backstage-plugins repo in the RoadieHQ organiztion.
```

Comment:
I suggest moving these out of "Steps" and into a new section called "Example configurations".

Then you can introduce the goal of each example config and use a code block to show the user what they would put into the input to replicate the example.